meson.build

```diff
@@ -88,7 +88,7 @@ endif
 vpd_manager_exe = executable(
                 'vpd-manager',
                 vpd_manager_SOURCES,
-                include_directories : ['include/', 'configuration/'],
+                include_directories : ['include/', 'configuration/', 'include/utility'],
```

Comment:
Don't need meson changes. Header can be included in respective cpp files directly as parent folder is already included.
`#include <utility/dbus_utility.hpp> will do.`

Anyways, I am ok either way.